eternalcombat-plugin/src/main/java/com/eternalcode/combat/border/BorderTriggerIndexBucket.java

```diff
@@ -0,0 +1,56 @@
+package com.eternalcode.combat.border;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+class BorderTriggerIndexBucket {
+
+    private final Map<Long, Set<BorderTrigger>> index = new HashMap<>();
+
+    private BorderTriggerIndexBucket() {
+    }
+
+    Set<BorderTrigger> getTriggers(int x, int z) {
+        long position = packChunk(x >> 8, z >> 8);
```

Comment:
What is 8?

Reply:
Dodane

Reply:
![image](https://github.com/user-attachments/assets/4cae2f97-f92c-4b81-b9f3-b5cfb7a7f5e2)